pkg/skaffold/schema/v2beta29/upgrade.go

```diff
@@ -300,3 +311,32 @@ func upgradePatches(olds []JSONPatch, news []next.JSONPatch) {
 		}
 	}
 }
+
+func mergeKustomizeIntoKubectlDeployer(newKubectlDeployer *next.KubectlDeploy, oldKustomizeDeployer *KustomizeDeploy, kubectlDeployerWasInjected bool) error {
```

Comment:
a good reading about variable names [here](go/go-style/decisions#variable-names)

Reply:
Awesome! Thanks for the reference, I changed the variables name. Let me know what do you think.